_format/1.1/index.md

```diff
@@ -794,21 +776,36 @@ document for that collection:
 }
 ```
 
-##### <a href="#document-links-link-relation-type" id="document-links-link-relation-type" class="headerlink"></a> Link relation type
+#### <a id="document-links-link-object"></a> Link objects
+
+A "link object" is an object that represents a [web link](https://tools.ietf.org/html/rfc8288).
 
-The value of the `rel` member **MUST** be a string that represents a link relation
-type. Valid link relation types are defined by
-[RFC8288 Section 2.1](https://tools.ietf.org/html/rfc8288#section-2.1).
+A link object **MUST** contain the following member:
 
-If a `rel` member is not given on a link object, the link's relation type
-**SHOULD** be inferred from the name of the link object.
+* `href`: a string whose value is a URI-reference [[RFC3986 Section 4.1](https://tools.ietf.org/html/rfc3986#section-4.1)]
+  pointing to the link's target.
 
-In order to represent a link with reversed semantics, it is **RECOMMENDED**
-that an alternate link relation type be used or, less preferably, that the
-`anchor` and `href` members be interchanged.
+A link object **MAY** also contain any of the following members:
 
-> Note: Historically, a `rev` link parameter was used for this purpose but has
-> since been deprecated by [RFC8288 Section 3.3](https://tools.ietf.org/html/rfc8288#section-3.3).
+* `rel`: a string indicating the link's relation type. The string **MUST** be a
+  [valid link relation type](https://tools.ietf.org/html/rfc8288#section-2.1).
```

Comment:
Did you intend to drop `anchor` ?

Reply:
Yes. I think it opens a can of worms WRT to URI fragments and "reversed" links. We can add it later if we really need it and we can fully flesh out how to linking to specific objects in a document.

Reply:
Seems reasonable 👍